token/service.go

```diff
@@ -38,7 +38,8 @@ func (s *tokenService) ResolveUserToken(ctx context.Context, target, token strin
 					Value: token,
 					Type:  "Bearer"}}})
 
-	res, err := client.RetrieveToken(ctx, authclient.RetrieveTokenPath(), target, nil)
+	forcePull := true
```

Comment:
We don't want to forcePull user tokens, only cluster tokens.

Reply:
ah, damn'it, I misunderstood :/ let me change that

Reply:
Oh, I misunderstood the requirement for this too. If you need to check the cluster (dsaas user) token and not the user tokens then force_pull is currently ignored for cluster tokens. But I've just created a pull request to add force_pull support for cluster tokens in the Auth service - https://github.com/fabric8-services/fabric8-auth/pull/351